jira/src/main/kotlin/gropius/sync/jira/config/IMSConfig.kt

```diff
@@ -14,7 +14,12 @@ import java.net.URI
  * @param imsTemplate the template of the current IMS
  */
 data class IMSConfig(
```

Comment:
missing documentation for new params

Reply:
ea53a37648d7db83a68035da94f96725bea10775